src/main/java/kostka/moviecatalog/service/ExternalCommentService.java

```diff
@@ -22,11 +24,22 @@ public ExternalCommentService(final RestTemplate restTemplate) {
         this.restTemplate = restTemplate;
     }
 
+    @HystrixCommand(fallbackMethod = "getCommentsFromCommentServiceFallback")
```

Comment:
Dotaz. Co když zadáš neexistujíci metodu například. Spadne to nějak? Dokáže to ten hystrix detekovat a vypsat rozumnou hlášku? Nebo to pak padá až při běhu s nějakým nesmyslem?

Reply:
ano, hystriw vypise exception ze FallbackDefinitionException: fallback method wasn't found: getCommentsFromCommentServiBLABLABLAceFallback([class java.lang.Long])] with root cause